core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20220627_add_submission_output_path.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="add_submission_output_path_column" author="mbemis" logicalFilePath="dummy">
+        <addColumn tableName="SUBMISSION">
+            <column name="OUTPUT_PATH" type="TEXT">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="backfill_submission_output_path_column" author="mbemis">
+        <sql stripComments="true">
+            UPDATE
+                SUBMISSION
+                INNER JOIN WORKSPACE ON SUBMISSION.WORKSPACE_ID = WORKSPACE.id
+            SET
+                SUBMISSION.OUTPUT_PATH = concat(
+                "gs://",
+                WORKSPACE.bucket_name,
+                "/",
+                LOWER(
+                    CONCAT(
+                        SUBSTR(HEX(SUBMISSION.ID), 1, 8), '-',
+                        SUBSTR(HEX(SUBMISSION.ID), 9, 4), '-',
+                        SUBSTR(HEX(SUBMISSION.ID), 13, 4), '-',
+                        SUBSTR(HEX(SUBMISSION.ID), 17, 4), '-',
+                        SUBSTR(HEX(SUBMISSION.ID), 21)
+                    )
+                )
```

Comment:
submission IDs are stored as UUID binary, not strings, so this unfortunate step is necessary to convert them into UUID strings